optuna_dashboard/ts/components/GraphHistory.tsx

```diff
@@ -1,373 +1,22 @@
-import React, { ChangeEvent, FC, useEffect, useState } from "react"
-import { useNavigate } from "react-router-dom"
+import { useTheme } from "@mui/material"
+import { PlotHistory } from "@optuna/react"
+import React, { FC } from "react"
 import { StudyDetail } from "ts/types/optuna"
-import { useConstants } from "../constantsProvider"
 import { usePlotlyColorTheme } from "../state"
 
-const plotDomId = "graph-history"
-
-interface HistoryPlotInfo {
-  study_name: string
-  trials: Optuna.Trial[]
-  directions: Optuna.StudyDirection[]
-  metric_names?: string[]
-}
-
 export const GraphHistory: FC<{
   studies: StudyDetail[]
   logScale: boolean
   includePruned: boolean
 }> = ({ studies, logScale, includePruned }) => {
-  const { url_prefix } = useConstants()
-
   const theme = useTheme()
   const colorTheme = usePlotlyColorTheme(theme.palette.mode)
-  const navigate = useNavigate()
-  const [xAxis, setXAxis] = useState<
-    "number" | "datetime_start" | "datetime_complete"
-  >("number")
-  const [markerSize, setMarkerSize] = useState<number>(5)
-
-  const [targets, selected, setTarget] =
-    useObjectiveAndUserAttrTargetsFromStudies(studies)
-
-  const trials = useFilteredTrialsFromStudies(
-    studies,
-    [selected],
-    !includePruned
-  )
-  const historyPlotInfos = studies.map((study, index) => {
-    const h: HistoryPlotInfo = {
-      study_name: study?.name,
-      trials: trials[index],
-      directions: study?.directions,
-      metric_names: study?.metric_names,
-    }
-    return h
-  })
-
-  useEffect(() => {
-    plotHistory(
-      historyPlotInfos,
-      selected,
-      xAxis,
-      logScale,
-      theme.palette.mode,
-      colorTheme,
-      markerSize
-    )
-    const element = document.getElementById(plotDomId)
-    if (element !== null && studies.length >= 1) {
-      // @ts-ignore
-      element.on("plotly_click", (data) => {
-        if (data.points[0].data.mode !== "lines") {
-          let studyId = 1
-          if (data.points[0].data.name.includes("Infeasible Trial of")) {
-            const studyInfo: { id: number; name: string }[] = []
-            studies.forEach((study) => {
-              studyInfo.push({ id: study.id, name: study.name })
-            })
-            const dataPointStudyName = data.points[0].data.name.replace(
-              "Infeasible Trial of ",
-              ""
-            )
-            const targetId = studyInfo.find(
-              (s) => s.name === dataPointStudyName
-            )?.id
-            if (targetId !== undefined) {
-              studyId = targetId
-            }
-          } else {
-            studyId = studies[Math.floor(data.points[0].curveNumber / 2)].id
-          }
-          navigate(
-            url_prefix +
-              `/studies/${studyId}/trials?numbers=${data.points[0].x}`
-          )
-        }
-      })
-      return () => {
-        // @ts-ignore
-        element.removeAllListeners("plotly_click")
-      }
```

Comment:
I think this part is missing in `tslib/react/src/components/PlotHistory.tsx`.

Reply:
I missed this part. Thank you.